config/src/config.rs

```diff
@@ -404,10 +404,18 @@ pub struct Mempool {
 #[derive(Deserialize, Serialize, Default, Debug, Clone, PartialEq, Eq)]
 #[serde(default)]
 pub struct Tapi {
-    /// Oppose WIP0020
-    pub oppose_wip0020: bool,
-    /// Oppose WIP0021
-    pub oppose_wip0021: bool,
+    /// Oppose WIP0022
+    pub oppose_wip0022: bool,
+    /// Oppose WIP0023
+    pub oppose_wip0023: bool,
```

Comment:
This is funny. WIP0023 is actually not implemented :thinking:

Reply:
F